### PR TITLE
[Merged by Bors] - feat(CategoryTheory): inductive construction of composable arrows

### PR DESCRIPTION
**Pull request by joelriou**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/ComposableArrows.lean [hunk 1]
```diff
@@ -257,6 +264,167 @@ lemma ext₁ {F G : ComposableArrows C 1}
...
 
+namespace Nerve
+
+variable {C}
+
+lemma δ₀_eq_composableArrows_δ₀ {n : ℕ}
+    (x : (nerve C).obj (Opposite.op (SimplexCategory.mk (n + 1)))) :
+    (nerve C).δ (0 : Fin (n + 2)) x = ComposableArrows.δ₀ x := rfl
+
+end Nerve
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/ComposableArrows.lean:428]: I wonder whether we should revert the import. If so, then we can do that in another PR.

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

File: Mathlib/CategoryTheory/ComposableArrows.lean (hunk 1 above)
Comment (joelriou) [reply Mathlib/CategoryTheory/ComposableArrows.lean:428]: @jcommelin  Do you mean have `Nerve` import `ComposableArrows` rather than the other direction, so that we may use `ComposableArrows` without relying on the whole simplicial stuff?

Comment (jcommelin) [reply Mathlib/CategoryTheory/ComposableArrows.lean:428]: Exactly! And it would also allow us to use `ComposableArrows` from the start in setting up API for `Nerve`.

**Comment by jcommelin** [comment]
bors d+

File: Mathlib/CategoryTheory/ComposableArrows.lean (hunk 1 above)
Comment (joelriou) [reply Mathlib/CategoryTheory/ComposableArrows.lean:428]: As bors failed, I shall do it in this PR.

Comment (joelriou) [reply Mathlib/CategoryTheory/ComposableArrows.lean]: This is done in https://github.com/leanprover-community/mathlib4/pull/8146/commits/6a9f2359fec047680e7558b306bbed6bec9ce083

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by ocfnash** [comment]
GitHub was flaking and bors crashed so we need to try this again

bors merge
